code/game/objects/items/weapons/handcuffs.dm

```diff
@@ -34,19 +35,41 @@
 			place_handcuffs(user, user)
 			return
 
-		var/can_place
+		var/fast_place
 		if(istype(user, /mob/living/silicon/robot))
-			can_place = TRUE
+			fast_place = TRUE
 		else
 			for (var/obj/item/grab/G in C.grabbed_by)
 				if (G.loc == user && G.state >= GRAB_AGGRESSIVE)
-					can_place = TRUE
+					fast_place = TRUE
 					break
 
-		if(can_place)
+		if(fast_place)
 			place_handcuffs(C, user)
 		else
-			to_chat(user, "<span class='danger'>You need to have a firm grip on [C] before you can put \the [src] on!</span>")
+			if(C.stat == DEAD) // gotta make sure those corpses don't run off...
+				restrain()
+				return
+			if(C.stat == UNCONSCIOUS)
+				restrain()
+				return
+			if(C.a_intent == I_HELP)
+				restrain() 
+			else
+				to_chat(user, "<span class='danger'>\the [C] is not unconcious or cooperative! Try getting a good grip on them first, or suggest they cooperate!</span>")
+				to_chat(C, "<span class='danger'>\the [user] is trying to restrain you, but you resist them! Try being more helpful if you meant to cooperate.</span>")
+
+/obj/item/handcuffs/proc/restrain(var/mob/living/carbon/C, var/mob/user)
+	if(is_restraining)
+		return
+	is_restraining = TRUE
+	to_chat(user, "<span class='danger'>You begin to restrain [C] with \the [src]. This would be faster if you had a firm grip on them!!</span>")
```

Comment:
```suggestion
	to_chat(user, "<span class='danger'>You begin to restrain [C] with \the [src]. This would be faster if you had a firm grip on them!</span>")
```